src/components/QueryOutput/QueryOutput.js

```diff
@@ -1793,7 +1803,6 @@ export default class QueryOutput extends React.Component {
           activeChartElementKey={this.props.activeChartElementKey}
           onLegendClick={this.onLegendClick}
           dataConfig={_cloneDeep(this.dataConfig)}
-          themeConfig={getThemeConfig(this.props.themeConfig)}
```

Comment:
why did you delete this line of code?

Reply:
It has two themeConfig={getThemeConfig(this.props.themeConfig)} by incident.
You can see there is another one above the line 1805